packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -100,8 +114,14 @@ class TransferEngine implements ITransferEngine {
 
   async transfer(): Promise<void> {
     try {
+      // Bootstrap the providers
       await this.boostrap();
```

Comment:
I just noticed this is spelled "boostrap"

Reply:
I'll fix it here

Reply:
Done